distribution/command-line/src/main/kotlin/org/codice/ckt/Command.kt

```diff
@@ -34,6 +36,10 @@ fun main(args: Array<String>) {
             listOf("plugins"),
             description = "Path to the plugins directory")
 
+    parser.flag("d",
+            listOf("debug"),
+            description = "Turn on debug logs.")
+
```

Comment:
❓ When running the `runsaml` script, do these logs get printed as well?

Reply:
If you provide the `-d` flag, yes